addons/binding/org.openhab.binding.onkyo/ESH-INF/config/config.xml

```diff
@@ -29,10 +29,16 @@
 		</parameter>
 		<parameter name="volumeLimit" type="integer" min="0" max="100">
 			<label>Volume Limit</label>
-			<description>Limit maximum volume level to defined value.</description>
+			<description>Limit maximum volume level to defined percentage.</description>
 			<default>100</default>
 			<advanced>true</advanced>
 		</parameter>
+		<parameter name="volumeScale" type="integer" min="0" max="100">
```

Comment:
Please apply the formatter in this file.

Reply:
Done